recipes/r-openssl/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.9.6' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-openssl
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: openssl_{{ version }}.tar.gz
```

Comment:
Thanks. This is what the skeleton proposes. I must say I kind of like this because it reflects that it is the unchanged original source. But if it is a policy I am fine with changing the skeleton.